app/services/clients/datacite.rb

```diff
@@ -84,12 +85,22 @@ def to_params
 
     # Query by affiliation
     class AffiliationQuery
-      def initialize(affiliation:)
+      def initialize(affiliation:, affiliation_id:)
         @affiliation = affiliation
+        @affiliation_id = affiliation_id
       end
 
       def to_params
-        { query: "creators.affiliation.name:\"#{@affiliation}\"" }
+        { query: "(creators.nameIdentifiers.nameIdentifier:\"#{@affiliation_id}\" OR " \
+                 "contributors.nameIdentifiers.nameIdentifier:\"#{@affiliation_id}\" OR " \
+                 "creators.name:\"#{@affiliation}\" OR " \
+                 "contributors.name:\"#{@affiliation}\" OR " \
+                 "creators.affiliation.name:\"#{@affiliation}\" OR " \
+                 "contributors.affiliation.name:\"#{@affiliation}\" OR " \
+                 "contributors.affiliation.affiliationIdentifier:\"#{@affiliation_id}\" OR " \
+                 "creators.affiliation.affiliationIdentifier:\"#{@affiliation_id}\")",
+          'types.resourceTypeGeneral': 'Dataset',
+          'fields[dois]': 'id,updated' }
```

Comment:
Note: The ticket suggested only returning `id`, however we use `updated` as the modifiedToken, so I kept that in.

Reply:
That is perfectly fine. Thank you

Reply:
Everything looks good in general. I know I put "resourceTypeGeneral:Dataset" in my example queries, but I believe this is already covered in the DataCite client: https://github.com/sul-dlss/dataworks-etl/blob/main/app/services/clients/datacite.rb#L62, so we can remove it from this query